packages/core-js-sdk/src/sdk/helpers/index.ts

```diff
@@ -32,6 +32,17 @@ export function isJwtExpired(token: string): boolean {
   return currentTime > exp;
 }
 
+/**
+ * Returns the list of tenants in the given JWT
+ *
+ * @param token JWT token
+ */
+export function getTenants(token: string): string[] {
+  let claims: any = parseJwt(token);
+  const items = Object.keys(claims?.tenants);
+  return Array.isArray(items) ? items : [];
```

Comment:
this logic of extracting the items from the key is already implemented in `getJwtAuthorizationItems` so we can utilize it, so I suggest to reuse it

```suggestion
  return getJwtAuthorizationItems(token, '', 'tenants');
```

Reply:
`tenants` is not an array and we need keys in this case. So will have to do some handling in the `getJwtAuthorizationItems` function right?

Reply:
could do something like
```
const items = claims?.[claim];
if (Array.isArray(items)) return items;
if (items instanceof Object) return Object.keys(items);
return []
```
in the `getJwtAuthorizationItems` function

Reply:
good point 👍 

thanks for the explantation